### PR TITLE
MOTECH-2730: Migrate Admin Topics UI

### PR DESCRIPTION
**Pull request by pkornowski**

### CODE REVIEW

File: src/admin/topics/topics.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<motech-list>
+    <motech-list-item ng-repeat="topic in topics" ng-show="dataAvailable">
+        <motech-list-column column-title="Name" sortable>{{topic.destination}}</motech-list-column>
+        <motech-list-column column-title="Consumer Count" sortable>{{topic.consumerCount}}</motech-list-column>
+        <motech-list-column column-title="Enqueue Count" sortable>{{topic.enqueueCount}}</motech-list-column>
+        <motech-list-column column-title="Dequeue Count" sortable>{{topic.dequeueCount}}</motech-list-column>
+        <motech-list-column column-title="Expiry Count" sortable>{{topic.expiredCount}}</motech-list-column>
+    </motech-list-item>
+    <motech-list-item ng-show="!dataAvailable" class="alert alert-danger">
+        {{msg('admin.topic.error')}}
```
Comment (jredlarski) [inline src/admin/topics/topics.html]: Please use translate: `{{'admin.topic.error' | translate}}`

**Comment by jredlarski** [comment]
@pkornowski After checking your PR I wonder if you could work a little bit more on the styling of the motech-list containing data about topics? I'm aware that motech-list directive requires some changes for better computing of the column sizes but in Scheduler module it looks better than in Topics so please take a look at it.

**Comment by pkornowski** [comment]
@jredlarski 
I looked at this more closely, and I think a problem is a motech-list directive. Width of the header is set by the widest cell of table  in a single column. In scheduler it looks better because  size of header content and cell below are similar. In Topics cell content is shorter generally, because it is a number.

**Comment by jredlarski** [comment]
@pkornowski Ok, I understand what is happening. I have created a ticket for this issue [on Jira](https://applab.atlassian.net/browse/MOTECH-2766).
Apart from that, I'm glad with your work.

**Comment by jredlarski** [comment]
I'm merging this one as there are only some layout issues which should be fixed by newly created ticket described [on Jira](https://applab.atlassian.net/browse/MOTECH-2766).

**Pull request merged**